project/ws/app/src/lib/routes/events/components/event-list-view/event-list-view.component.ts

```diff
@@ -79,6 +85,13 @@ export class EventListViewComponent implements OnInit, AfterViewInit, OnChanges
     } else {
       this.dataSource.filter = ''
     }
+    this.events.raiseInteractTelemetry(
+      'click',
+      'card-cardContent',
+      {
+        id: this.configSvc.userProfile.userId,
```

Comment:
Logged in user id shout not go here, it's wrong. either pass the object what u r interacting with its  id or anything that is relevant to object on which interact is happening. Change it in every place